docs/dev_setup.rst

```diff
@@ -0,0 +1,88 @@
+.. Developer Setup for libCellML
+
+=================
+Development Setup
+=================
+
+This section describes how someone wanting to contribute to the libCellML project should set up their *working copy* for developing libCellML.
+
+.. contents::
+
+Overview
+========
+
+The libCellML codebase is hosted on GitHub and therefore Git is used to track changes in the codebase.  Before you begin you will need to have a few prerequisites satisfied:
+
+#. Github user account (for the rest of this document we will call our user *andre*)
+#. Git installed
+
+This document doesn't cover the process of satisfying these prerequisites, it is left as an exercise for you (Google is your friend).  The remainder of this document assumes that these prerequisites have been met.  This document covers setup from the command line if you are using a GUI like 'GitHub for Windows' then you will need to adjust the commands for the GUI you are using.
+
+The goal here is to get a working copy of source code, tests, and documentation onto your PC so that you can begin development.  To make this happen you will need to fork the :term:`prime libCellML repository`, make a clone onto your PC, and set up the Git remotes.  In :numref:`fig_devSetup_githubRepos` you can see a pictorial representation of what we are aiming to achieve.
+
+.. _fig_devSetup_githubRepos:
+
+.. figure:: images/libCellMLProcesses-GitHubRepos.png
```

Comment:
Shouldn't the **prime (read only)** dashed arrow between the **CellML** and **PC** 'boxes' be going from **CellML** to **PC** and not the other way round?